src/utils/testing/mocks/mockTag.ts

```diff
@@ -0,0 +1,23 @@
+import { mockObject } from 'utils/testing/mocks';
+import { ZetkinTag } from 'types/zetkin';
+
+const tag: ZetkinTag = {
+  color: null,
+  description: 'People who organize',
+  group: null,
+  hidden: false,
+  id: 1,
+  organization: {
```

Comment:
Extract this out into "mockOrganization.ts"? There's a couple of other files in the /utils/testing/mocks folder with an organization object...

Reply:
That's a good point